### ISSUE TITLE
Code is not fully compatible with PSReadLine v2.0

### ISSUE THREAD

**Issue by ForNeVeR** [JanDeDobbeleer/oh-my-posh2#67]
According to [an advice from PSReadLine author](https://github.com/lzybkr/PSReadLine/issues/555#issuecomment-356845616), I've installed PSReadLine v2.0.0-beta1 to resolve issue on PowerShell Core. After that, I see the following errors while loading my profile:

```
Set-PSReadLineOption : A parameter cannot be found that matches parameter name 'TokenKind'.
At E:\Projects\oh-my-posh\defaults.ps1:76 char:22
+ Set-PSReadlineOption -TokenKind Command -ForegroundColor DarkBlue
+                      ~~~~~~~~~~
+ CategoryInfo          : InvalidArgument: (:) [Set-PSReadLineOption], ParameterBindingException
+ FullyQualifiedErrorId : NamedParameterNotFound,Microsoft.PowerShell.SetPSReadLineOption

Set-PSReadLineOption : A parameter cannot be found that matches parameter name 'TokenKind'.
At E:\Projects\oh-my-posh\defaults.ps1:77 char:22
+ Set-PSReadlineOption -TokenKind Parameter -ForegroundColor Yellow
+                      ~~~~~~~~~~
+ CategoryInfo          : InvalidArgument: (:) [Set-PSReadLineOption], ParameterBindingException
+ FullyQualifiedErrorId : NamedParameterNotFound,Microsoft.PowerShell.SetPSReadLineOption
```

**Comment by nwykes** [comment]
A quick fix is to replace the following in the bottom of defaults.ps1

```powershell
Set-PSReadlineOption -TokenKind Command -ForegroundColor DarkBlue
Set-PSReadlineOption -TokenKind Parameter -ForegroundColor Yellow
```

with 

```powershell
Set-PSReadlineOption -Colors @{
    "Command" = [ConsoleColor]::DarkBlue
    "Parameter" = [ConsoleColor]::Yellow
}
```

It'll probably need to do some kind of version checking though since the new -Colors is only supported on the 2.0 beta.

This (meaning PSReadline 2.0) also fixes #64 on powershell core 6.0.

**Comment by JanDeDobbeleer** [comment]
This is one that will need a version check, I'm currently writing a PS Core compatible version that will a also need to tackle this one.
